scripts/db_migrator.py

```diff
@@ -670,6 +670,24 @@ def migrate_config_db_flex_counter_delay_status(self):
                 flex_counter['FLEX_COUNTER_DELAY_STATUS'] = 'true'
                 self.configDB.mod_entry('FLEX_COUNTER_TABLE', obj, flex_counter)
 
+    def migrate_sflow_table(self):
+        """
+        Handle sflow table migration. Migrations handled:
+        'sample_direction' attr in SFLOW object was newly introduced.
+        Set defaults 'sample_direction = rx' on upgrade.
+        """
+        sflow_table = self.appDB.get_table("SFLOW_TABLE")
+        for key, value in sflow_table.items():
+            if 'sample_direction' not in value:
+                sflow_key = "SFLOW_TABLE:{}".format(key)
+                self.appDB.set(self.appDB.APPL_DB, sflow_key, 'sample_direction','rx')
```

Comment:
Why is the degault value set to `rx`. Is this default present in minigraph.xml / minigraph.py?
If yes, let's consume the default value from `self.minigraph_data`, to avoid hardcoding here?

Reply:
Default is not part of minigraph. Retaining explicit hard-coded value for better code readability.

Reply:
The concern w/ hardcoding on migration path is that it may lead to unexpected config in future if the source of truth (for eg., minigraph) changes.

We have had such issues in the past and I intend to avoid that by not duplicating hardcoded values and importing from a one source of truth.

Does minigraph parser not generate this `rx` value? If there is no way this value can be consumed from a config source in the target image then the current solution is acceptable.

Reply:
sflow configs are not part of minigraph and there is no config source that we know of.

Reply:
db_migrator changes are removed from this PR and will be raised as separate PR.